students/gyeongjae-ham/westagram/users/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from django import forms
+
+from phone_field import PhoneField
+
+class User(models.Model):
+    name = models.CharField(max_length=45)
+    email = models.EmailField(max_length=200)
+    password = models.CharField(max_length=45)
+    number = PhoneField()
```

Comment:
'='을 기준으로 정렬해 주세요. 위코드 파이썬 코드 컨벤션 입니다.
```suggestion
    name     = models.CharField(max_length=45)
    email    = models.EmailField(max_length=200)
    password = models.CharField(max_length=45)
    number   = PhoneField()
```